environment-api/src/main/java/com/sequenceiq/environment/api/v1/environment/model/response/DetailedEnvironmentResponse.java

```diff
@@ -87,6 +87,8 @@ public void setProxyConfig(ProxyResponse proxyConfig) {
 
         private AwsEnvironmentParameters aws;
 
+        private AzureEnvironmentParametersResponse azure;
```

Comment:
AzureEnvironmentParametersResponse -> AzureEnvironmentParameters

Reply:
Ok, and can AzureEnvironmentParametersRequest stay the same?